contract/models/res_partner.py

```diff
@@ -8,35 +8,47 @@ class ResPartner(models.Model):
     _inherit = 'res.partner'
 
     sale_contract_count = fields.Integer(
-        string='Sale Contracts',
-        compute='_compute_contract_count',
+        string='Sale Contracts', compute='_compute_contract_count'
     )
     purchase_contract_count = fields.Integer(
-        string='Purchase Contracts',
-        compute='_compute_contract_count',
+        string='Purchase Contracts', compute='_compute_contract_count'
     )
 
     def _compute_contract_count(self):
         contract_model = self.env['account.analytic.account']
         today = fields.Date.today()
-        fetch_data = contract_model.read_group([
-            ('recurring_invoices', '=', True),
-            ('partner_id', 'child_of', self.ids),
-            '|',
-            ('date_end', '=', False),
-            ('date_end', '>=', today)],
-            ['partner_id', 'contract_type'], ['partner_id', 'contract_type'],
-            lazy=False)
-        result = [[data['partner_id'][0], data['contract_type'],
-                   data['__count']] for data in fetch_data]
+        fetch_data = contract_model.read_group(
+            [
+                ('recurring_invoices', '=', True),
+                ('partner_id', 'child_of', self.ids),
+                '|',
+                ('date_end', '=', False),
+                ('date_end', '>=', today),
+            ],
+            ['partner_id', 'contract_type'],
+            ['partner_id', 'contract_type'],
+            lazy=False,
+        )
+        result = [
+            [data['partner_id'][0], data['contract_type'], data['__count']]
+            for data in fetch_data
+        ]
         for partner in self:
             partner_child_ids = partner.child_ids.ids + partner.ids
-            partner.sale_contract_count = sum([
-                r[2] for r in result
-                if r[0] in partner_child_ids and r[1] == 'sale'])
-            partner.purchase_contract_count = sum([
-                r[2] for r in result
-                if r[0] in partner_child_ids and r[1] == 'purchase'])
+            partner.sale_contract_count = sum(
+                [
```

Comment:
I don't understand why you need to do this kind of cosmetics changes. This creates more difference, adds difficulty to trace the originator of these lines and there is not really any visual improvement.

Reply:
Sorry about that, I delegate code formatting to [balck](https://github.com/ambv/black) and it's automatically make this "cosmetics changes". I will try to perform a revert on this kind of changes to facilitate code review.